src/Bridge/RefreshTokenRepository.php

```diff
@@ -4,18 +4,19 @@
 
 use Illuminate\Database\Connection;
 use Illuminate\Contracts\Events\Dispatcher;
+use Laravel\Passport\RefreshTokenRepository as RTokenRepository;
```

Comment:
I'm not happy with this naming here, would `TokenRepository` be more appropriate?